R/bainCommon.R

```diff
@@ -1107,200 +1173,178 @@
-    
-    groupCol <- dataset[ , options[["fixedFactors"]]]
+    groupCol <- dataset[, options[["fixedFactors"]]]
     varLevels <- levels(groupCol)
     bainResult <- bainContainer[["bainResult"]]$object
     bainSummary <- summary(bainResult, ci = options[["credibleInterval"]])
-    
-    if (type == "ancova")
+
+    if (type == "ancova") {
       bainSummary <- bainSummary[1:length(varLevels), ]
-    
+    }
+
     variable <- bainSummary[["Parameter"]]
     N <- bainSummary[["n"]]
     mu <- bainSummary[["Estimate"]]
     CiLower <- bainSummary[["lb"]]
     CiUpper <- bainSummary[["ub"]]
     yBreaks <- jaspGraphs::getPrettyAxisBreaks(pretty(c(CiLower, CiUpper)), min.n = 4)
     plotData <- data.frame(v = variable, N = N, mean = mu, lowerCI = CiLower, upperCI = CiUpper, index = 1:length(variable))
-    p <- ggplot2::ggplot(plotData, ggplot2::aes(x=index, y=mean)) +
-      ggplot2::geom_errorbar(ggplot2::aes(ymin=lowerCI, ymax=upperCI), colour="black", width=.2, position = ggplot2::position_dodge(.2)) +
-      ggplot2::geom_line(position=ggplot2::position_dodge(.2), size = .7) +
-      ggplot2::geom_point(position=ggplot2::position_dodge(.2), size=4) +
-      ggplot2::scale_fill_manual(values = c(rep(c("white","black"),5),rep("grey",100)), guide=ggplot2::guide_legend(nrow=10)) +
-      ggplot2::scale_shape_manual(values = c(rep(c(21:25),each=2),21:25,7:14,33:112), guide=ggplot2::guide_legend(nrow=10)) +
-      ggplot2::scale_color_manual(values = rep("black",200), guide=ggplot2::guide_legend(nrow=10)) +
-      ggplot2::scale_y_continuous(name = options[["dependent"]], breaks = yBreaks) +
-      ggplot2::scale_x_continuous(name = options[["fixedFactors"]], breaks = 1:length(varLevels), labels = as.character(varLevels))
-    p <- jaspGraphs::themeJasp(p)
+    p <- ggplot2::ggplot(plotData, ggplot2::aes(x = index, y = mean)) +
+      ggplot2::geom_errorbar(mapping = ggplot2::aes(ymin = lowerCI, ymax = upperCI), color = "black", width = 0.2) +
+      jaspGraphs::geom_line() +
+      jaspGraphs::geom_point(size = 4) +
+      #   ggplot2::scale_fill_manual(values = c(rep(c("white","black"),5),rep("grey",100)), guide=ggplot2::guide_legend(nrow=10)) +
+      #   ggplot2::scale_shape_manual(values = c(rep(c(21:25),each=2),21:25,7:14,33:112), guide=ggplot2::guide_legend(nrow=10)) +
+      #   ggplot2::scale_color_manual(values = rep("black",200), guide=ggplot2::guide_legend(nrow=10)) +
+      ggplot2::scale_y_continuous(name = options[["dependent"]], breaks = yBreaks, limits = range(yBreaks)) +
+      ggplot2::scale_x_continuous(name = options[["fixedFactors"]], breaks = 1:length(varLevels), labels = varLevels) +
+      jaspGraphs::geom_rangeframe() +
+      jaspGraphs::themeJaspRaw()
     plot$plotObject <- p
   }
 }
 
 .plotModelProbabilitiesTTests <- function(x, type) {
-  
   if (type == 1 || type == 2 || type == 3) {
-    labs <- c(gettext("H0"), gettext("H1"))
+    labels <- c(gettext("H0"), gettext("H1"))
```

Comment:
```suggestion
    labels <- gettext(c("H0", "H1"))
```

this should work as well, if you prefer to reduce the number of times you write out `gettext` (there are other lines where this occurs as well). But I'll leave that up to you whether you want to change it

Reply:
I've adjusted this but will leave a thorough investigation of this in the module for another time.